cpp/src/slice2cs/Gen.cpp

```diff
@@ -66,20 +66,16 @@ isDefaultInitialized(const MemberPtr& member, bool considerDefaultValue)
 string
 opFormatTypeToString(const OperationPtr& op)
 {
-    // TODO: eliminate DefaultFormat in the parser (DefaultFormat means the communicator default that was removed in
-    // Ice 4.0)
-    switch (op->format())
+    if (auto format = op->format())
```

Comment:
maybe we can get rid of this if and use `value_or`

```
    switch (op->format().value_or(CompactFormat))
    {
        case CompactFormat: return "ZeroC.Ice.FormatType.Compact";
        case SlicedFormat:  return "ZeroC.Ice.FormatType.Sliced";
    }

Reply:
Yes, I agree! But it'll only be doable for C#, since that's the only language where we've actually made `Compact` the default.

Reply:
given than there are just two values

```
return op->format().value_or(CompactFormat) == CompactFormat ?
    "ZeroC.Ice.FormatType.Compact" : "ZeroC.Ice.FormatType.Sliced";
```